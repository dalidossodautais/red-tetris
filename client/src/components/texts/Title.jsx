import styled from "styled-components";

export default styled.h2.attrs((props) => ({
  color: props.theme.primaryTextColor,
}))`
  color: ${(props) => props.color};
  fontweight: ${(props) => props.fontWeight};
  text-overflow: ellipsis;
`;
