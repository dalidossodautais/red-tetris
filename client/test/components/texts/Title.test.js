import React from "react";
import renderer from "react-test-renderer";
import Title from "../../../src/components/texts/Title";

describe("Title", () => {
  test("Standard", () => {
    const component = renderer.create(<Title />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
