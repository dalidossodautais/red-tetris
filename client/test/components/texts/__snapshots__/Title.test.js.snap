// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Standard 1`] = `
<h2
  className="sc-bdfBwQ fbxoca"
/>
`;

exports[`Title Standard 2`] = `
<h2
  className="sc-bdfBwQ fbxoca"
/>
`;
